### PR TITLE
Webserver | login.html now uses logos, favicon added 

### PR DESCRIPTION
**Pull request by reggietorres**

### CODE REVIEW

**Comment by reggietorres** [comment]
don't mind the unverified, linux broke and in recovery i lost the private key

File: templates/index.html [hunk 1]
```diff
@@ -75,6 +43,51 @@
...
-        
+
+        <div class="fixed bottom-0 right-0 m-4 bg-red-500 text-white p-4 rounded shadow-lg" id="logoutWarning" style="display: none;">
+            <p>You will be logged out in 5 seconds due to inactivity. Click the button below to extend your session.</p>
+            <button onclick="extendSession()" class="mt-2 bg-green-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded">Extend Session</button>
+        </div>
+        <!-- ... -->
+        <script>
+            var logoutWarningTimeout;
+            var logoutTimeout = 5 * 1000; // 5 seconds in milliseconds
```
Comment (A-c0rN) [inline templates/index.html:54]: I would suggest lengthening this, most users wouldn't recognize nor read a prompt in 5 seconds.
Normally, these are 30 to 60 seconds.

File: README.md [hunk 2]
```diff
@@ -123,6 +123,8 @@ CREATE TABLE users (
 ```
 then, go in web.py and change line 21 to the information you filled out for your table..
 
+go to line 23 in web.py and generate a **VERY** long secret key!
```
Comment (A-c0rN) [inline README.md:126]: If we can automate this, that would be great! I have a code block to generate a random secret key brick.

File: web.py [hunk 3]
```diff
@@ -5,14 +5,14 @@
...
-from flask import Flask, render_template, request, session, redirect, url_for
+from flask import Flask, render_template, request, session, redirect, url_for, send_from_directory
 from flask_sqlalchemy import SQLAlchemy
 from pytz import utc
 from discord_webhook import DiscordWebhook, DiscordEmbed
 
 
 #Here we define the webhook
-webhook = DiscordWebhook(url="webhook here", username="ASMARA 2FA", avatar_url="https://avatars.githubusercontent.com/u/20572623?v=4")
+webhook = DiscordWebhook(url="webhook here", username="ASMARA 2FA", avatar_url="https://github.com/A-c0rN/ASMARA/blob/main/assets/asmara-logo-only.png?raw=true")
```
Comment (A-c0rN) [inline web.py:15]: I would highly suggest NOT using discord for 2FA, I should have seen this last time.
Not every user has Discord, nor is it in any way secure. 
I would suggest to skip 2FA until we have a way to support Email, Phone, and/or Physical 2FA

File: templates/index.html (hunk 1 above)
Comment (reggietorres) [reply templates/index.html:54]: Workin on it

File: README.md (hunk 2 above)
Comment (reggietorres) [reply README.md:126]: Working on it

Comment (reggietorres) [reply README.md:126]: Will be included in latest commit

File: web.py (hunk 3 above)
Comment (reggietorres) [reply web.py:15]: For now, functionality will be deactivated.. I need sleep.

**Review by A-c0rN — APPROVED** [review]
Looks good for now,
I am going to experiment with doing key generation in pure python, and push it up later.
Maybe also storing everything in Environment Vars so we're not doing a buncha disk read/writes.
Thank you!

**Pull request merged**